### DISCUSSION TITLE
Hotkey Instructions

### DISCUSSION THREAD

**Discussion by feyyd** [EveOPlus/eve-o-preview#16] [General]
In the instructions on the main pages README, it is advised to change the 

`"ClientHotkey": {},`

to

```
"ClientHotkey": {
  "EVE - Phrynohyas Tig-Rah": "F1",
  "EVE - Ondatra Patrouette": "Control+Shift+F4"
}
```

The removal of the comma will cause a crash on start of Eve-O-Preview which my be unclear how to fix to a novice user.

**Comment by EveOPlus** [comment]
This is now obsolete. I'll need to update the readme.md but client hotkeys are now managed through a single toon cycle group to simplify the setup and maintenance.
